node/src/eventsbuilder.ts

```diff
@@ -7,11 +7,11 @@
 import { SeekNextCommit, SeekPosition, SeekSpecified } from './protos/orderer/ab_pb';
 
 /**
- * Options used when requesting events.
+ * Options used when requesting chaincode events.
```

Comment:
These options are shared by chaincode and block eventing. They are not specific to chaincode events.